packages/features/bookings/Booker/components/BookEventForm/BookingFields.tsx

```diff
@@ -59,6 +62,16 @@ export const BookingFields = ({
           // rescheduleReason is a reschedule specific field and thus should be editable during reschedule
           readOnly = false;
         }
+        if (field.name === SystemField.Enum.hostAssignment) {
+          // Check if bookingData is null or if schedulingType is defined and not ROUND_ROBIN
+          if (bookingData === null || (schedulingType && schedulingType !== SchedulingType.ROUND_ROBIN)) {
+            return null;
+          }
+          if (!getValues("responses.hostAssignment")) {
+            setValue(`responses.${SystemField.Enum.hostAssignment}`, "Round-Robin host");
```

Comment:
It would be better to use Enum instead of using "Round-robin host" and "Same host" directly.